test/e2e/support/utils.js

```diff
@@ -0,0 +1,69 @@
+import path from 'path';
+import url from 'url';
+
+const BASE_URL = 'http://localhost:8888';
```

Comment:
It was possible to provide a different value for `BASE_URL`, `USERNAME` and `PASSWORD`. We should look into it.

Reply:
See: `cypress_base_url=http://my-custom-basee-url cypress_username=myusername cypress_password=mypassword npm run test-e2e`

Reply:
yes, this was available by default in cypress, we need to do it ourselves in the puppeteer setup but not that hard I think.

Reply:
Yes, shouldn't be an issue. Can be done later.

Reply:
Could just be some simple env variables:

```js
const { E2E_BASE_URL = 'http://localhost:8888' } = process.env;
```

Reply:
There was this idea someone shared, that we might want to introduce `wordpress` section inside `package.json` to provide a way to override some variables.

```json
{
    "wordpress": {
        "baseUrl": "http://localhost:8888"
    }
}
```